core/storage/suggestion/gae_models_test.py

```diff
@@ -1948,3 +1948,721 @@ def test_export_data_nontrivial(self) -> None:
             .export_data(self.CONTRIBUTOR_USER_ID))
 
         self.assertEqual(expected_data, user_data)
+
+
+class TranslationReviewStatsModelUnitTests(test_utils.GenericTestBase):
```

Comment:
Please add a test for the deletion policy. Ditto elsewhere.

Reply:
Shown below.